Mage.Sets/src/mage/cards/z/ZinniaValleysVoice.java

```diff
@@ -0,0 +1,152 @@
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterCreatureCard;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.common.FilterNonlandCard;
+import mage.filter.predicate.mageobject.AnotherPredicate;
+import mage.filter.predicate.mageobject.BasePowerPredicate;
+import mage.cards.Card;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.game.Game;
+import mage.game.stack.Spell;
+import mage.players.Player;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+import static mage.abilities.dynamicvalue.common.StaticValue.*;
+import static mage.constants.Duration.WhileOnBattlefield;
+
+/**
+ *
+ * @author DreamWaker
+ */
+public final class ZinniaValleysVoice extends CardImpl {
+
+    static final FilterCreaturePermanent bfilter = new FilterCreaturePermanent("other creatures with base power 1");
+
+    static {
+        bfilter.add(new BasePowerPredicate(ComparisonType.EQUAL_TO, 1));
+        bfilter.add(TargetController.YOU.getControllerPredicate());
+        bfilter.add(AnotherPredicate.instance);
+
+    }
+
+    static final PermanentsOnBattlefieldCount bcount = new PermanentsOnBattlefieldCount(bfilter);;
+
+    static final FilterNonlandCard cfilter = new FilterNonlandCard("creature spells");
+
+    static {
+        cfilter.add(CardType.CREATURE.getPredicate());
+        cfilter.add(AnotherPredicate.instance);
+    }
+
+    public ZinniaValleysVoice(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[] { CardType.CREATURE }, "{U}{R}{W}");
+
+        this.supertype.add(SuperType.LEGENDARY);
+        this.subtype.add(SubType.BIRD);
+        this.subtype.add(SubType.BARD);
+        this.power = new MageInt(1);
+        this.toughness = new MageInt(3);
+
+	    // Flying
+        this.addAbility(FlyingAbility.getInstance());
+
+        // Creature spells you cast have offspring 2.
+        this.addAbility(
+                new SimpleStaticAbility(new GainAbilityControlledSpellsEffect(new OffspringAbility("{2}"), cfilter)
+                        .setText("Creature spells you cast have offspring {2}.")));
+        this.addAbility(new SimpleStaticAbility(new GainAbilityControlledEffect(
+                new OffspringAbility("{2}"), Duration.WhileOnBattlefield,
+                StaticFilters.FILTER_CREATURES_NON_TOKEN, true))
+        //        new SimpleStaticAbility(new ZinniaValleysVoiceGainOffspringEffect())
+        );
+
+	    // Zinnia Valley's Voice gets +X/+0, where X is the number of other creatures you control with base power 1.
+        this.addAbility(new SimpleStaticAbility(new BoostSourceEffect(bcount, get(0), WhileOnBattlefield)));
+
+    }
+
+    private ZinniaValleysVoice(final ZinniaValleysVoice card) {
+        super(card);
+    }
+
+    @Override
+    public ZinniaValleysVoice copy() {
+        return new ZinniaValleysVoice(this);
+    }
+}
+
+class ZinniaValleysVoiceGainOffspringEffect extends ContinuousEffectImpl {
```

Comment:
if not used, don't include it

Reply:
Its so other devs can work with it to see if a better fix can be found. As it is, the card does work as intended in almost all cases.